### DISCUSSION TITLE
화면 공유 표시를 잘 보이게 건의

### DISCUSSION THREAD

**Discussion by neopandora** [metamong-friends/metamong-friends.github.io#43] [제안하기]
화면 공유를 하는데 얼굴 부분에 작게 표시가 되어서 어디를 눌러야 공유된 화면이 보이는지 못찾는 경우가 있어서 좀 더 잘보이게 크기나 모양이 수정되면 좋겠습니다.

**Comment by ACT-TJ** [comment]
안녕하세요!

사용자 들의 캐릭터, 비디오 및 기타 아이콘들이 Zoom in - out (마우스 스크롤 동작) 에 따라 상대적으로 크기가 가변적이다 보니 멀리서 보는 view 로 맞춰놓았을 때 아이콘이 잘 눈에 띄지 않는 상황이 있는 것 같습니다. 또한 스크린쉐어 아이콘은 그 중에서도 작게 표현이 되니까요.

해당 부분 가시성 개선 건으로 관리하도록 하겠습니다.  현재는 유저 비디오 영역을 특정 최소값에서 더 작아지지 않는 방향으로 poc 를 진행하고 있습니다. 의미있는 변경안이 반영되면 릴리즈 공지를 드리도록 하겠습니다.

감사합니다!

**Comment by shlee303** [comment]
@neopandora 줌아웃 해도 일정 크기가 유지하도록 조치하였습니다. 좋은 의견 감사합니다! 혹시 아직 잘 안보이신다고 생각되면 의견 주시면 아이콘 자체를 좀 더 수정해보겠습니다!
